cloud_functions/climateiq_spatialize_chunk_predictions_cf/main.py

```diff
@@ -289,7 +289,7 @@ def _build_spatialized_model_predictions(
     )
 
 
-def _add_h3_index_details(cell: pd.Series) -> pd.Series:
+def _add_h3_index_details(cell: pd.Series, chunk_boundary: Any) -> pd.Series:
```

Comment:
this probably isn't actually an `Any` ?
you can do
```
print(chunk_boundary.__class__)
```
if you aren't sure what the right type is